pkg/s3/s3.go

```diff
@@ -90,18 +91,19 @@ func CreateS3BucketUsingAPod(ctx context.Context, clientset kubernetes.Interface
 	createBucketPodOutput := CreateBucketPodOutput{}
 
 	scanner := bufio.NewScanner(bytes.NewReader(logs))
+	var logError error
 	for scanner.Scan() {
 		line := scanner.Text()
 
-		if err := json.Unmarshal([]byte(line), &createBucketPodOutput); err != nil {
+		if logError = json.Unmarshal([]byte(line), &createBucketPodOutput); logError != nil {
 			continue
 		}
 
 		break
 	}
 
-	if !createBucketPodOutput.Success {
-		return errors.Wrapf(err, "failed to create bucket, please check %s pod logs for more details", createBucketPod.Name)
+	if !createBucketPodOutput.Success || logError != nil {
```

Comment:
won't `logError` always be NOT nil? since we're echoing invalid json in the script and unmarshalling these messages will fail?